pkg/api/v1alpha1/tinkerbelldatacenterconfig_types.go

```diff
@@ -26,6 +26,13 @@ type TinkerbellDatacenterConfigSpec struct {
 	SkipLoadBalancerDeployment bool `json:"skipLoadBalancerDeployment,omitempty"`
 	// LoadBalancerInterface can be used to configure a load balancer interface for the Tinkerbell stack.
 	LoadBalancerInterface string `json:"loadBalancerInterface,omitempty"`
+	// IsoBoot can be used to indicate that the hardware must boot using an ISO.
+	//+optional
+	IsoBoot bool `json:"isoBoot,omitempty"`
+	// HookIsoURL is the URL of ISO image that will one time boot.
```

Comment:
`one time boot` is indeed accurate but is this how you would describe this to a user? is there other ways to describe the ISO, like being used during the provisioning process, etc.